samples/Extensions/Table/TableFunction.cs

```diff
@@ -4,28 +4,26 @@
 using System;
 using Microsoft.Azure.Functions.Worker;
 using Microsoft.Extensions.Logging;
-using Newtonsoft.Json.Linq;
 
 namespace SampleApp
 {
     public static class TableFunction
     {
         [Function("TableFunction")]
-        [TableOutput("outputQueue", Connection = "ServiceBusConnection")]
+        [TableOutput("OutputTable", Connection = "AzureWebJobsStorage")]
         public static MyTableData Run([QueueTrigger("table-items")] string input,
-            [TableInput("MyTable", "MyPartition", "{queueTrigger}")] JObject tableItem,
```

Comment:
@ankitkumarr was this here to demonstrate the ability to use a `JObject` in this context?